spark/src/main/resources/python/zeppelin_pyspark.py

```diff
@@ -107,6 +107,50 @@ def isAutoConvertEnabled(self):
   def isImportAllPackageUnderSparkSql(self):
     return self.version >= self.SPARK_1_3_0
 
+class PySparkCompletion:
+  def getGlobalCompletion(self):
+    objectDefList = []
+    try:
+      for completionItem in list(globals().iterkeys()):
+        objectDefList.append(completionItem)
+    except:
+      return None
+    else:
+      return objectDefList
+
+  def getMethodCompletion(self, text_value):
+    objectDefList = []
+    completion_target = text_value
+    try:
+      if len(completion_target) <= 0:
+        return None
+      if text_value[-1] == ".":
```

Comment:
I understand.
I think the good feedback.
And by that omission in the auto-complete because it should not call,
I hope to add a filtering operation for __.
Thank you for feedback.